subprojects/core/src/integTest/groovy/org/gradle/internal/classpath/BuildScriptClasspathIntegrationSpec.groovy

```diff
@@ -242,21 +242,21 @@ class BuildScriptClasspathIntegrationSpec extends AbstractIntegrationSpec implem
         succeeds("showBuildscript")
 
         then:
-        def jar = inJarCache("proj.jar").assertExists()
+        def buildscriptClasses = inJarCache("proj/").assertExists()
```

Comment:
💅 Should we rename also test to "cleans up unused cached **entries** in the Jars cache"?